topology/platform/common/ssp.m4

```diff
@@ -37,6 +37,12 @@ define(`SSP_CONFIG_DATA',
 `	tuples."short" {'
 `		SOF_TKN_INTEL_SSP_MCLK_ID'	ifelse($4, `', "0", STR($4))
 `	}'
+`	tuples."short" {'
+`		SOF_TKN_INTEL_SSP_FRAME_PULSE_WIDTH'	ifelse($5, `', "0", STR($5))
```

Comment:
Can we combine the two short type tokens within the same tuples."short" {} array?

Reply:
If we combine them together, how the token parser to process them in kernel side?
maybe we need add more code in the sof_link_ssp_load() function to do this.
I think this would cause the code too complicated to read.

Reply:
I wonder if we could put the frame width in the fs_sync configuration? It'd be more logical.

Reply:
I will think about it. the PR in the kernel will be updated accordingly.

Reply:
@plbossart 
I did not find the fs_sync configuration, you mean the SSP_CLOCK(fsync, ....)?
what about put the parameter "frame_pulse_width" in SSP_TDM() in ssp.m4 file?
bit clock and fs clock both use the SSP_CLOCK() macro.

Reply:
@plbossart 
I checked this, the alsa_lib did not support this.
we have to use this in the token at the moment.
we could update the alsa_lib in future to support this parameter.

Reply:
Yes, putting the frame_pulse_width in the SSP_TDM macro would make sense, but I am not sure it's feasible since it's a new token.
You may want to keep it in SSP_CONFIG_DATA but you want to reorder the parameters so that we get mclk_id, quirks, frame_pulse_width. The latter makes no sense for I2S and LeftJ so we want to have it last, while quirks could be common to all modes

Reply:
@plbossart 
OK, I will update the order of the parameters.